app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -192,7 +194,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.addFunding(userId, body)
+        val actualResponse = apiService.addFunding(bearerToken, userId, body)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver